### PR TITLE
Updating units direction is now configurable.

### PR DESCRIPTION
**Pull request by csnv**
Added update_direction as bitmask option in battle/client.conf, defaults
to 0 (no updates at all).

### CODE REVIEW

**Comment by MishimaHaruna** [comment]
Any reason why it defaults to 0? Is that what it happens on official servers? (considering it's different from our behavior before merging this)

File: src/map/battle.c [hunk 1]
```diff
@@ -7258,6 +7258,7 @@ static const struct battle_data {
 	{ "min_body_style",                     &battle_config.min_body_style,                  0,      0,      SHRT_MAX,       },
 	{ "max_body_style",                     &battle_config.max_body_style,                  4,      0,      SHRT_MAX,       },
 	{ "save_body_style",                    &battle_config.save_body_style,                 0,      0,      1,              },
+	{ "update_direction",                   &battle_config.update_direction,               BL_NUL,  BL_NUL, BL_ALL,         },
```
Comment (dastgir) [inline src/map/battle.c]: Default should be BC_ALL, isn't it? (Same applies to client.conf)

**Comment by csnv** [comment]
No info from official server was considered when this feature was introduced, that's why it should default to 0.

If somebody can test it in an Aegis server would be great, but otherwise and due to lack of info, I would let it as it was before this feature was introduced.

**Comment by Smokexyz** [comment]
This shouldn't be an optional thing. All unit's directions should be updated by default.

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1126)

**Pull request closed without merging**